packaging/docker/README.md

```diff
@@ -49,33 +49,51 @@ services:
     volumes:
       - /proc:/host/proc:ro
       - /sys:/host/sys:ro
-      - /var/run/docker.sock:/var/run/docker.sock:ro
```

Comment:
There is one more place above that refers to the socket.
Sorry i missed it on the first sweep